common/main/aa_DJ.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
-<!-- Copyright © 1991-2022 Unicode, Inc.
+<!-- Copyright © 1991-2023 Unicode, Inc.
```

Comment:
I'd been not checking in these copyright-only changes. Would be good to skip these, regardless of how CLDR-14239 shakes out.